rules/no-empty-file.js

```diff
@@ -17,9 +17,9 @@ const hasTripeSlashDirectives = comments =>
 
 /** @param {import('eslint').Rule.RuleContext} context */
 const create = context => {
-	const filename = context.getPhysicalFilename().toLowerCase();
+	const filename = context.getPhysicalFilename();
 
-	if (!/\.(?:js|mjs|cjs|ts|mts|cts)$/.test(filename)) {
+	if (!/\.(?:js|mjs|cjs|jsx|ts|mts|cts|tsx)$/i.test(filename)) {
```

Comment:
```suggestion
	if (!/\.(?:[mc]?[jt]sx?)$/i.test(filename)) {
```

Would support stuff like `mjsx` too.

Reply:
I prefer current pattern, it's more readable and won't match `mjsx`.

Reply:
Both are equally readable to me, but whatever you prefer. I think `mjsx` is inevitable.

Reply:
I prefer the current pattern too.